config/projects.yml

```diff
@@ -80,6 +80,13 @@ classification-bundle:
         symfony: ['2.3', '2.7', '2.8', '3.0', '3.1']
         sonata_admin: ['3']
 
+classification-media-bundle:
+  branches:
+    master:
+      php: ['5.6', '7.0']
```

Comment:
Didn't take a look to the dependence separation, but it would be great to have a BC way.

In this case, we should support older dependencies, shouldn't we?